ocs_ci/ocs/ui/views.py

```diff
@@ -33,6 +33,10 @@
     "ocp_page": f"Overview · {azure_managed}Red Hat OpenShift",
     "login_page_title": "Log in · Red Hat OpenShift",
 }
+# Bug opened in Jira https://issues.redhat.com/browse/OCPBUGS-15419. Tmp solution to check locators
```

Comment:
This jira is verified. Do we need to revert this change?

Reply:
Let's do it then

Reply:
> Let's do it then

fixed, thanks